docs/Android.md

```diff
@@ -67,144 +61,29 @@ The steps are as follows:
-
-Create `armeabi-v7a` directory and generate `icu` directory by uncompressing `icu4c-55_1-src.tgz` file, as following:
- 
-```
-$ mkdir armeabi-v7a
-$ cd armeabi-v7a
-$ tar xvf ../icu4c-55_1-src.tgz
-```
-
-**Edit icu configure file**
-
-In order to prevent icu configuration from adding `swift` suffix to internal symbols as well, edit `icu/source/configure` file and change:
-
-```
-if test "$ICULIBSUFFIX" != ""
-then
-    U_HAVE_LIB_SUFFIX=1
-    ICULIBSUFFIXCNAME=`echo _$ICULIBSUFFIX | sed 's/^A-Za-z0-9_/_/g'`
-    UCONFIG_CPPFLAGS="${UCONFIG_CPPFLAGS} -DU_HAVE_LIB_SUFFIX=1 -DU_LIB_SUFFIX_C_NAME=${ICULIBSUFFIXCNAME} "
-else
-    U_HAVE_LIB_SUFFIX=0
-fi
-```
-
-To:
- 
-```
-#if test "$ICULIBSUFFIX" != ""
-#then
-#    U_HAVE_LIB_SUFFIX=1
-#    ICULIBSUFFIXCNAME=`echo _$ICULIBSUFFIX | sed 's/^A-Za-z0-9_/_/g'`
-#    UCONFIG_CPPFLAGS="${UCONFIG_CPPFLAGS} -DU_HAVE_LIB_SUFFIX=1 -DU_LIB_SUFFIX_C_NAME=${ICULIBSUFFIXCNAME} "
-#else
-    U_HAVE_LIB_SUFFIX=0
-#fi
-```
-
-**Edit build.sh file**
-
-Now go back to the `libiconv-libicu-android` root directory:
-
-```
-$ cd ..
-```
-
-In order to prevent `build.sh` from regenerating the icu directory and override the `configure` file we just edited, edit `build.sh` and change:
-
-```
-[ -e ../icu4c-55_1-src.tgz ] || exit 1
-tar xvf ../icu4c-55_1-src.tgz
-```
-
-To:
-
-```
-#[ -e ../icu4c-55_1-src.tgz ] || exit 1
-#tar xvf ../icu4c-55_1-src.tgz
-```
-
-In order to check the existence of `libicuucswift.so` file instead of `libicuuc.so`, change:
-
-```
-[ -e libicuuc.so ] || {
-```
-
-To:
-
-```
-[ -e libicuucswift.so ] || {
-```
-
-Then add the `swift` suffix to the call of `configure` file, by changing:
-
-```
-./configure \
---host=arm-linux-androideabi \
---prefix=`pwd`/../../ \
---with-cross-build=`pwd`/cross \
---enable-static --enable-shared \
-|| exit 1
-```
-To:
-
-```
-./configure \
---host=arm-linux-androideabi \
---prefix=`pwd`/../../ \
---with-library-suffix=swift \
---with-cross-build=`pwd`/cross \
---enable-static --enable-shared \
-|| exit 1
-```
-
-Now change the libraries file names on the last step of th build, by adding `swift` at the end of each one, by changing:
-
-```
-for f in libicudata libicutest libicui18n libicuio libicule libiculx libicutu libicuuc; do
-```
-
-To:
-
-```
-for f in libicudataswift libicutestswift libicui18nswift libicuioswift libiculeswift libiculxswift libicutuswift libicuucswift; do
-```
-
-
-**Run build.sh**
-
-Now we are finally ready to run the `build.sh` file
-
-```
-$ ./build.sh
-```
-
-Confirm that the build script created `armeabi-v7a/icu/source/i18n` and
-   `armeabi-v7a/icu/source/common` directories within your
-   `libiconv-libicu-android` directory.
+4. Change directories into `libiconv-libicu-android`: `cd libiconv-libicu-android`
+5. Run the Swift build script: `./build-swift.sh`
```

Comment:
Unless this changed in the libiconv-libicu-android repo, you'll need an env-var `LIBSUFFIX="swift"` for the files to be generated as the rest of the document assumes they're generated.

Reply:
Ah, ok, that makes sense. Ignore this then. Are those changes published somewhere?

Reply:
Sorry for the wait @lanza, I granted you access to that repository.